Makefile

```diff
@@ -22,3 +24,8 @@ uninstall:
 		/lib/modules/*/extra/hid-viewsonic.ko
 	udevadm control --reload
 	depmod -a
+
+distdir = $(PACKAGE)-$(VERSION)
```

Comment:
Could you perhaps name this variable `PACKAGE`, and put it above with the other two?